__init__.py

```diff
@@ -193,138 +210,146 @@ def on_websettings_changed(self):
-        info_ready_path = join(self.pianobar_path, 'info_ready')
+        info_ready_path = join(self.pianobar_path, "info_ready")
         if isfile(info_ready_path):
             self._load_current_info()
             try:
                 remove(info_ready_path)
             except Exception as e:
-                LOG.error(e)
+                self.log.error(e)
 
             # Update the "Now Playing song"
-            LOG.info("State: "+str(self.piano_bar_state))
+            self.log.info("State: " + str(self.piano_bar_state))
             if self.piano_bar_state == "playing":
-                self.enclosure.mouth_text(self.settings["song_artist"] + ": " +
-                                          self.settings["song_title"])
+                self.enclosure.mouth_text(
+                    self.play_info["song_artist"] + ": " + self.play_info["song_title"]
+                )
 
     def cmd(self, s):
         self.process.stdin.write(s.encode())
         self.process.stdin.flush()
 
     def troubleshoot_auth_error(self):
-        user_info = get_pandora_user_info(self.settings['email'],
-                                          self.settings['password'])
+        user_info = get_pandora_user_info(
+            self.settings["email"], self.settings["password"]
+        )
         if user_info:
-            if user_info.get('stationCount') == 0:
-                self.speak_dialog('no.stations')
+            if user_info.get("stationCount") == 0:
+                self.speak_dialog("no.stations")
             else:
-                self.speak_dialog('pandora.error')
+                self.speak_dialog("pandora.error")
         else:
-            self.speak_dialog('wrong.credentials')
+            self.speak_dialog("wrong.credentials")
 
     def _init_pianobar(self):
-        if self.settings.get('first_init') is False:
+        if self.play_info.get("first_init") is False:
             return
 
         # Run this exactly one time to prepare pianobar for usage
         # by Mycroft.
         try:
-            LOG.info("INIT PIANOBAR")
+            self.log.info("INIT PIANOBAR")
             subprocess.call(["killall", "-9", "pianobar"])
-            self.process = subprocess.Popen(["pianobar"],
-                                            stdin=subprocess.PIPE,
-                                            stdout=subprocess.PIPE)
+            self.process = subprocess.Popen(
+                ["pianobar"], stdin=subprocess.PIPE, stdout=subprocess.PIPE
+            )
             time.sleep(3)
             self.cmd("0\n")
             self.cmd("S")
             time.sleep(0.5)
             self.process.kill()
-            self.settings['first_init'] = False
+            self.play_info["first_init"] = False
             self._load_current_info()
         except Exception as e:
-            LOG.exception('Failed to connect to Pandora: '+repr(e))
+            self.log.exception("Failed to connect to Pandora: " + repr(e))
```

Comment:
You don't need the `repr(e)` in a log.exception as it is automatically included in the log message.

Reply:
Good spot, fixed.